### PR TITLE
feat(presentation): add main title layout

### PR DESCRIPTION
**Pull request by Gijuno**

### CODE REVIEW

**Comment by Gijuno** [comment]
additional works after #40 and #41 close

File: app/src/main/res/layout/layout_main_title.xml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <TextView
+        android:textColor="@color/text_black"
+        android:id="@+id/subtitle_textview"
+        android:textSize="14sp"
+        android:fontFamily="@font/noto_sans_kr_regular"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        android:text="반갑습니다"
```
Comment (nutyworks) [inline app/src/main/res/layout/layout_main_title.xml:16]: Please fix the hard-coded string.

File: app/src/main/res/layout/layout_main_title.xml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"/>
+
+    <TextView
+        android:textColor="@color/text_black"
+        android:text="사용할 기능을 선택하세요"
```
Comment (nutyworks) [inline app/src/main/res/layout/layout_main_title.xml:22]: Please fix the hard-coded string.

File: app/src/main/res/layout/layout_main_title.xml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"/>
+
+    <ImageView
+        app:layout_constraintBottom_toBottomOf="@id/title_textview"
+        app:layout_constraintTop_toTopOf="@id/subtitle_textview"
+        app:layout_constraintEnd_toEndOf="parent"
+        android:src="@drawable/splash_logo"
+        android:layout_width="45dp"
+        android:layout_height="45dp"/>
```
Comment (nutyworks) [inline app/src/main/res/layout/layout_main_title.xml:37]: `contentDescription` attribute is missing. Please consider adding it for better accessibility.

**Review by nutyworks — CHANGES_REQUESTED** [review]
Please reformat the layout file.
* There should be a space before `/>`.
* XML attributes should be aligned in alphabetical order.

**Review by nutyworks — APPROVED** [review]

**Pull request merged**